src/XMakeTasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1602,7 +1602,7 @@ private void LogAssembliesConsideredAndRejected(Reference reference, MessageImpo
                                 break;
                             }
                         case NoMatchReason.FusionNamesDidNotMatch:
-                            Log.LogMessageFromResources(importance, "ResolveAssemblyReference.EightSpaceIndent", Log.FormatResourceString("ResolveAssemblyReference.ConsideredAndRejectedBecauseFusionNamesDidntMatch", location.FileNameAttempted, location.AssemblyName.FullName));
+                            Log.LogMessageFromResources(importance, "ResolveAssemblyReference.EightSpaceIndent", Log.FormatResourceString("ResolveAssemblyReference.ConsideredAndRejectedBecauseFusionNamesDidntMatch", location.FileNameAttempted, location.AssemblyName.FullName, location.AssemblyName.Name));
```

Comment:
The new argument, expected name, should probably be `fusionName` available in the caller (`LogReference`). I couldn't see how that might be available via the `reference` argument itself though.